packages/venia-ui/lib/components/ProductOptions/tile.module.css

```diff
@@ -25,3 +26,20 @@
 .root_selected_focused {
     composes: root_selected;
 }
+
+.root_outOfStock,
+.root_selected_outOfStock {
+    composes: root;
+    background: #f5f5f5;
+    opacity: 0.55;
+    border: 2px solid rgb(var(--venia-global-color-gray-400));
+    composes: text-gray-600 from global;
+}
+.root_outOfStock:after,
+.root_selected_outOfStock:after {
+    content: '';
+    position: absolute;
+    border-top: 2px solid rgb(var(--venia-global-color-gray-400));
+    width: 62px;
+    transform: rotate(-45deg);
+}
```

Comment:
See if you can replace some of these values with Tailwind utilities

Reply:
Thanks! it's updated